webservices/schemas.py

```diff
@@ -405,6 +435,14 @@ class CandidateHistoryTotalSchema(schemas['CandidateHistorySchema'], schemas['Ca
             'contributor_employer_text',
             'contributor_occupation_text',
         ),
+        'relationships': [
```

Comment:
As mentioned above, the query gets reformed during the export code to scale back to just the model itself, and this is a field that is on a related/nested model.  Explicitly looking at this information (and in order to get it, we ultimately need the actual column to query against and the label in which we'd like it to be called).

At the point of where this is processed, we're only dealing with the schema `Meta` information and the model itself.  Adding this `relationships` component to the `Meta` object allows us to explicitly define these things on an as-needed basis without having it impact the existing infrastructure/API output.  `marshmallow` and `flask` no nothing about this object except in the export code, which explicitly looks for it.

Since the query is re-written to be just the fields on the primary model we're dealing with (in this case, the itemized schedule data), there are no joins to begin with.  Any relationship added here (and the export code checks to see if the relationship between tables has already been added so it doesn't add it more than once) will only add that specific join.

In the end, all of this code only adds a single join between the schedule table being exported and the committee history associated with a record (based on a `committee_id`).  The query ends up being a lot simpler then the API endpoint query. :-)